config/header-snippets/types_legacy.h.in

```diff
@@ -85,9 +85,14 @@ typedef signed long long duk_int64_t;
     (defined(DUK_F_MINGW) || defined(DUK_F_MSVC))
 /* Both MinGW and MSVC have a 64-bit type. */
 #define DUK_F_HAVE_64BIT
+#if defined(DUK_F_MSVC)
+typedef unsigned __int64 duk_uint64_t;
```

Comment:
Quite sure, unfortunately, as I have its help file open right now. I'm not sure when they introduced `long long` but it didn't exact with 6 (which is probably also the reason they don't support the LL suffix, but require the i64 one).